.circleci/config.yml

```diff
@@ -1429,10 +1445,7 @@ jobs:
         only_for_branches: master
         webhook: '${SLACK_WEBHOOK_KERNEL_PROBE_BUILD_FAILURE}'
 
-workflows:
-  version: 2
-  build:
-    jobs:
+reusejobs: &reusejobs
```

Comment:
The name for this anchor is not very descriptive, can we try to rename it to something like `fullWorkflow` or `completeWorkflow`?

Reply:
Done